src/components/ContextNav/ContextNav.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import NewTheme from '../../themes/new';
+
+const ContextNavWrapper = styled.div`
+  display: flex;
+  background: ${(props) => props.bgColor};
+  border-bottom: 1px solid ${(props) => props.theme.colors.whiteTone};
+  justify-content: center;
+  position: relative;
+  a {
+    position: relative;
+    padding: 18px 8px;
+    margin: 0 8px;
+    font-size: 13px;
+    line-height: 21px;
+    color: ${(props) => props.color};
+    text-decoration: none;
+    border: none;
+    white-space: nowrap;
+    text-transform: uppercase;
+    &:last-of-type {
+      margin-right: 0;
+      padding-right: 16px;
+    }
+    &:hover {
+      color: ${(props) => props.color};
+    }
+    &.active::before,
+    &:hover::before {
+      position: absolute;
+      content: '';
+      height: 4px;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      background-color: ${(props) => props.ascent};
```

Comment:
Don't you mean `accent`? IMO, being a prop it should be suffixed with `Color`, like `accentColor`. That way its intent is clear

Reply:
Yeap, you're right. I meant accent. `accentColor` sounds better. Thanks.